lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.html

```diff
@@ -1,10 +1,12 @@
 <label class="adf-property-label"
      [attr.data-automation-id]="'card-dateitem-label-' + property.key"
+     [ngClass]="{'adf-property-label-not-editable' : !isEditable() && editable}"
```

Comment:
that does not make any sense... "not editable && editable" is the opposite... are you sure it's working properly? extract to a single function/getter and provide tests for that